modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,149 @@
+= Migrate Client Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: migrate client applications, migrate applications, migrate 
+
+You might have client credentials of applications residing outside of Anypoint Platform if you were previously using a different API management. You must first migrate those client applications, with their client ID and client secret credentials, before you can start using Anypoint Platform.
+
+Because client IDs are unique in Anypoint Platform, you cannot reuse the same client ID in two different accounts. Your client applications can consume APIs whose organization is registered with a Anypoint Platform client provider or an external client provider. The instructions for these two scenarios differ:
+
+* <<migrate_with_internal,Migrate client applications for APIs with Anypoint Platform client providers>>
+* <<migrate_with_external,Migrate client applications for APIs with external client Providers>>
+
+== Prerequisites
+
+Before migrating the client application, ensure that you:
+
+* Obtain the required token and credentials:
+** Client ID and client secret for the client application to migrate
+** Anypoint Platform https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/pages/Authentication/[token]
+* Review the <<considerations,scope and limitations>> to understand the limitations and scope of the migration task
+
+[[considerations]]
+== Scope and Limitations
+
+Before you migrate your client applications to Anypoint Platform, review the following scope and limitations: 
+
+* If the organization of the API you consume has registered an external client provider, for example, PingFederate or Okta, review the section: <<external_client,Organizations with Registered External Client Providers>>.
+* Do not attempt to update the client ID and client secret of an application.
++
+This feature is not supported.
+* Do not use the same user to create more than 500 applications.
+* Do not create applications with duplicate names.
++
+Even if the identity provider (IdP) stores the duplicate applications with different client ID and client secret credentials for the applications, issues can occur.
+* Do not attempt to refresh the client ID.
++
+Refreshing the client ID is not possible unless you delete and re-register the client application.
+* If the client credentials you provide match with the credentials of an existing organization ID on the client provider, the application gets associated with that client.
++
+Conversely, If the client credentials that you provide do not match, the request does not get associated with the client and fails.
+
+[[migrate_with_internal]]
+== Migrate Client Applications for APIs with Anypoint Platform Client Providers
+
+To migrate your client application:
+
+. Open a text editor and paste the following commands:
```

Comment:
Alternative to having two separate steps and a Note and a final step that belongs HERE:
```suggestion
. Open a text editor and paste and run the following commands, replacing `orgId` with the ID of your master organization and the content of the body with the appropriate application <<properties,properties>>:
```

Reply:
Otherwise, use original but add "run"; the final step should not be running the command from the command line, because that is implicit in using a text editor and automatic after inputting the changes.
```suggestion
. Open a text editor and paste and run the following commands:
```